### ISSUE TITLE
Gilroy Font Family

### ISSUE THREAD

**Issue by Avinashk1231** [fluttercommunity/community#148]
When I try to use the Gilroy Font Family, it doesn't appear the same as it does on the Figma screens. However, fonts from Google Fonts display correctly. My question is, can we use the Gilroy Font in Flutter Application, and if so, how?

**Comment by ibrahimsezer** [comment]
Yes, you can use Gilroy Font Family in your Flutter application. However, since gilroy is a special font, some adjustments are required. Here are some points to follow in general:

Gilroy Font Family contains multiple weights and styles (for example, Light, Regular, Bold, Italic). The font weight/style used in Figma may be missing in your Flutter project. You should check this.

Figma's rendering process and flutter work differently. Therefore, different images may appear at some points.

You should get it by checking the licence, we know that gilroy is a special font. It will be useful for you to use it by checking the licence status.

Download all the weights and styles of Gilroy Font Family you need (for example, Gilroy-Light.ttf, Gilroy-Bold.ttf, Gilroy-Italic.ttf).

Create a fonts folder in the root directory of your project (for example, assets/fonts/).
Copy the downloaded Gilroy .ttf or .otf files to this folder.

Example directory structure:

assets/
  fonts/
    Gilroy-Light.ttf
    Gilroy-Regular.ttf
    Gilroy-Bold.ttf
    Gilroy-Italic.ttf


Configure Pubspec.yaml File
Edit the pubspec.yaml file to introduce fonts to Flutter. Specify the font family and the correct file for each weight/style. A sample configuration:

flutter:
  fonts:
    - family: Gilroy
      fonts:
        - asset: assets/fonts/Gilroy-Light.ttf
          weight: 300
        - asset: assets/fonts/Gilroy-Regular.ttf
          weight: 400
        - asset: assets/fonts/Gilroy-Bold.ttf
          weight: 700
        - asset: assets/fonts/Gilroy-Italic.ttf
          style: italic

Default Font Setting: If you want to use Gilroy as the default font for the entire application, you can set fontFamily in ThemeData:

For example:
`MaterialApp(
  theme: ThemeData(
    fontFamily: ‘Gilroy’,
  ),
  home: MyHomePage(),
)`
